packages/client-core/src/world/components/editor/EditorContainer.tsx

```diff
@@ -159,7 +159,7 @@ class EditorContainer extends Component<EditorContainerProps, EditorContainerSta
       } else if (queryParams.has("sceneId")) {
         this.loadScene(queryParams.get("sceneId"));
       } else {
-        this.loadProjectTemplate(defaultTemplateUrl);
+        //this.loadProjectTemplate(defaultTemplateUrl);
```

Comment:
is this left over from testing? what's the reason this is commented out?

Reply:
Ya sorry..
It was for testing